### PR TITLE
update code based on feedback

### PR DESCRIPTION
**Pull request by michael-alade**
## Feedback

- No console logs
- Install eslint then extend the airbnb linting guide
- Line 56: Typo error
- Line 214: Typo error
- Get middle method test if the length of the string is odd.
- @return param should specify the type.

### CODE REVIEW

**Comment by andela-kakpobome** [comment]
The description for this is missing, please provide context by outlining the feedback that was implemented.

**Comment by michael-alade** [comment]
Please, could you clarify? @andela-kakpobome

**Comment by andela-kakpobome** [comment]
Clarify how? What is not clear exactly?

**Comment by michael-alade** [comment]
"Outlining the feedback that was implemented." @andela-kakpobome

**Comment by andela-kakpobome** [comment]
This PR states that it implements changes based on some feedback, meanwhile the description of this PR does not have the feedback outlined/described.

**Comment by michael-alade** [comment]
Ok thanks @andela-kakpobome

**Comment by michael-alade** [comment]
Updated @andela-kakpobome

File: public/src/string.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * 
  * Checks a string for any vowels
  * 
- * @return 
+ * @return {boolen}
```
Comment (andela-kakpobome) [inline public/src/string.js:9]: ### Nitpick

Spellcheck this

File: public/src/string.js [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
  */
 String.prototype.hasVowels = function() {
```
Comment (andela-kakpobome) [inline public/src/string.js:11]: ### Nitpick

This violates the airbnb standard.

File: public/src/string.js [hunk 3]
```diff
@@ -104,19 +104,19 @@ String.prototype.toCurrency = function () {
...
- * Returns a number representation
- * of the Currency String
+ * Returns each letter in the string
+ * as an inverse of its current case
  * 
- * @return
+ * @return {string}
  */
 String.prototype.inverseCase = function () {
   return this.replace(/\w/g, (match) => {
```
Comment (andela-kakpobome) [inline public/src/string.js:122]: ### Critical

Line 125 violates the airbnb standard

Comment (michael-alade) [reply public/src/string.js:122]: All nitpicks has been resovled in #27  @andela-kakpobome

**Pull request merged**